### ISSUE TITLE
Edge Orientation Is Wrong In Some Cases When Center Is At Origin

### ISSUE THREAD

**Issue by jmwright** [7BIndustries/Semblage#41]
An edge with the following endpoints is obviously not supposed to be oriented along the Z as it is in the screenshot.
```
(0, 0, 0)
(0, 0, 40)
```
![Screenshot from 2021-05-24 16-21-19](https://user-images.githubusercontent.com/1015439/119405978-1ce57d80-bcb0-11eb-8fe9-1f2910118d22.png)

The source of this (Python-side or GDScript-side) needs to be tracked down.

**Comment by jmwright** [comment]
The following Semblage component code exhibits the same behavior with an incorrectly oriented edge at the origin. The edge is incorrectly oriented both before and after the extrude step.
```python
# Semblage v0.2.0-alpha
import cadquery as cq
# start_params
# end_params
result=cq
result=result.Workplane("XY").workplane(invert=True,centerOption="CenterOfBoundBox").tag("Change")
result=result.moveTo(-37.5,0).hLine(75,forConstruction=False).vLine(10,forConstruction=False).threePointArc(point1=(10,20),point2=(5,20),forConstruction=False).hLine(-10,forConstruction=False).threePointArc(point1=(-32.5,13),point2=(-37.5,10),forConstruction=False).close()
result=result.extrude(10,combine=True,clean=True,both=False,taper=0.0)
result=result.faces("<Y").workplane(invert=False,centerOption="CenterOfBoundBox")
result=result.circle(1.0,forConstruction=False)
result=result.cutThruAll(clean=True,taper=0.0)

show_object(result)
```
![Screenshot from 2021-06-01 09-38-25](https://user-images.githubusercontent.com/1015439/120332866-2462e300-c2bd-11eb-80bf-c1750fddf6ed.png)

**Comment by jmwright** [comment]
Fixed in 2fe7f1160393087cea6cdc8365c2a99f542cfdb4

![Screenshot from 2021-06-01 17-59-26](https://user-images.githubusercontent.com/1015439/120396201-2734f680-c304-11eb-9dcc-04934a8fb7ca.png)

![Screenshot from 2021-06-01 18-01-32](https://user-images.githubusercontent.com/1015439/120396214-2c924100-c304-11eb-8a09-0b7a9d7e483e.png)
